Description: skip browser libraries build
 This needs to many dependencies
Author: Pirate Praveen <praveen@debian.org>
Forwarded: not-needed
Reviewed-By: Yadd <yadd@debian.org>
Last-Update: 2019-12-14

--- a/scripts/build.js
+++ b/scripts/build.js
@@ -189,6 +189,6 @@
   packages.forEach(buildNodePackage);
   process.stdout.write('\n');
 
-  process.stdout.write(chalk.inverse(' Building browser packages \n'));
-  packages.forEach(buildBrowserPackage);
+  //process.stdout.write(chalk.inverse(' Building browser packages \n'));
+  //packages.forEach(buildBrowserPackage);
 }
